docs/manifest.json

```diff
@@ -929,6 +929,12 @@
 		"markdown_source": "../packages/components/src/radio-control/README.md",
 		"parent": "components"
 	},
+	{
+		"title": "RadioGroup",
```

Comment:
Interesting, it’s experimental but we expose it in developer documentation. I hope to have a look into it and figure out how we can improve it. The good part is that https://developer.wordpress.org/block-editor/components/ is generated from the branch that targets WP 5.4 😃